src/main/java/ingest/utility/IngestUtilities.java

```diff
@@ -120,6 +120,8 @@
 	private String AMAZONS3_BUCKET_NAME;
 	@Value("${vcap.services.pz-blobstore.credentials.encryption_key}")
 	private String S3_KMS_CMK_ID;
+	@Value("${s3.use.kms}")
+	private Boolean USE_KMS;
```

Comment:
Does this need to be the Boolean class for some reason?  Then we pass it in to getAwsClient which takes boolean... is that a problem?

Reply:
Maybe i'm reading this wrong